diagnostic.md

```diff
@@ -149,9 +174,11 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+divOne will be at the top left
+divTwo will be below on the left
+divThree will be below on the left
+divFour will be on the same line as divThree but flush right
+divFive will be below divFour on the left
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5